src/xeus_sqlite_interpreter.cpp

```diff
@@ -69,24 +90,28 @@ void interpreter::load_db(const std::vector<std::string> tokenized_code)
     {
         if (tokenized_code[2] == tokenized_code.back())
         {
-            this->db = std::make_unique<SQLite::Database>(tokenized_code[2],
+            this->m_db = std::make_unique<SQLite::Database>(tokenized_code[2],
```

Comment:
no `this->` needed.

Reply:
why is this resolved? I still see a `this->` :)

Reply:
hahaha sorry
I mark them as I fix them, like a check list. I'll commit the changes soon.